scripts/disabled_tests/tests/data/playlist_out.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "JDK_VERSION": "all",
+    "PLATFORM": "all",
+    "ISSUE_TRACKER": "https://github.com/eclipse-openj9/openj9/issues/11904",
+    "TARGET": "jdk_custom",
+    "CUSTOM_TARGET": "ParallelStreamsLoadTest_special_J9_23",
+    "JDK_IMPL": "hotspot"
+  },
+  {
+    "JDK_VERSION": "all",
+    "PLATFORM": "all",
+    "ISSUE_TRACKER": "https://github.com/eclipse-openj9/openj9/issues/11904",
+    "TARGET": "jdk_custom",
+    "CUSTOM_TARGET": "ParallelStreamsLoadTest_special_J9_24",
+    "JDK_IMPL": "ibm"
+  },
+  {
+    "JDK_VERSION": "8+",
+    "PLATFORM": "all",
+    "ISSUE_TRACKER": "https://github.com/eclipse-openj9/openj9/issues/11904",
+    "TARGET": "jdk_custom",
+    "CUSTOM_TARGET": "ParallelStreamsLoadTest_special_J9_25",
+    "JDK_IMPL": "hotspot"
+  },
+  {
+    "JDK_VERSION": "11",
+    "PLATFORM": "all",
+    "ISSUE_TRACKER": "some non functional comment",
+    "TARGET": "jdk_custom",
+    "CUSTOM_TARGET": "ParallelStreamsLoadTest_special_J9_0",
+    "JDK_IMPL": "openj9"
+  }
+]
```

Comment:
I saw you have a bunch of `<platform>.*windows.*</platform>` in playlist_input.xml, but I  couldn't find matching `"PLATFORM": ".*windows.*"`. Is there anything I missed?

Reply:
Oops, I'm sorry for the late reply, I didn't see the comment.

Unfortunately, this is a somewhat undocumented decision that was made in a weekly meeting. The main issue stems from the fact that
a) the parsers don't carry information about all concrete platforms which we support (thus, we can't directly apply the regex and keep only applicable platforms); and
b) the `PLATFORM` field is intended to map to a runnable platform in Jenkins (which the regex expressions aren't).

Of course, both of those points can be changed. The current decision was to always output "ALL", launch it on Jenkins, and then let each TKG instance determine whether or not it should run the test.

Reply:
For the record, during code reviews of playlists, the reviewers are not advising people to put the full platform name in most cases.  There will still be always be regex though, and yes, thanks for the reminder that @llxia advised in one of our weekly meetings to launch 'all' when in doubt.  :)

Reply:
> to launch 'all' **when in doubt** (_emphasis mine_)

Just to clarify: at the moment, that the parser is set to always output "ALL"; we don't try to parse the platform by any means. Should I?

As reference, below are all the values from the `<platform>` nodes and their occurrences.
```
'aarch64_mac.*', 30
'x86-64_windows', 16
'x86-64_mac', 14
'.*windows.*', 14
'.*zos.*', 12
'x86-64_mac|aarch64_mac', 9
'x86-64_mac|ppc64_aix', 6
'x86-64_windows|x86-32_windows', 6
'.*mac.*', 6
'x86-32_windows|arm_linux', 6
'x86-32_windows', 5
'.*aix.*', 4
'ppc64_aix', 2
'x86-64_windows|x86-64_mac', 2
's390x_zos', 2
'aarch64.*', 2
'.*aarch64.*', 1
'x86-64_alpine-linux', 1
'.*x86-64_solaris.*', 1
'^((?!(x86-64_linux|x86-64_mac|x86-64_windows)).)*$', 1
'x86-64_solaris|sparcv9_solaris', 1
'ppc64le.*', 1
'aarch64.*mac.*', 1
'.*aarch.*', 1
```